subsys/bluetooth/controller/ll_sw/pdu.h

```diff
@@ -527,9 +527,11 @@ struct pdu_data {
 
 	u8_t len;
 
+#if !defined(CONFIG_SOC_OPENISA_RV32M1_RISCV32)
 #if !defined(CONFIG_BT_CTLR_DATA_LENGTH_CLEAR)
 	u8_t resv:8; /* TODO: remove nRF specific code */
 #endif /* !CONFIG_BT_CTLR_DATA_LENGTH_CLEAR */
+#endif /* !CONFIG_SOC_OPENISA_RV32M1_RISCV32 */
```

Comment:
Maybe the TODO should be resolved instead of piling up #ifdefs?

Reply:
Agree with the removal. The reason it is still here is that we will probably use a similarly-named field when implementing LL encryption. But for now it makes no sense to keep it.

Reply:
This field is required by nRF platforms. Instead of the member, a macro can be placed that is replaced with this field by nordic's vendor headers. But this should be a separate PR task, and not to pollute this PR.

Reply:
@cvinayak , so I will leave these #defines as-is and send a new PR which implements the proposed solution for this field. Is this OK with you?

Reply:
@alexandru-porosanu-nxp yes, ok with me.